documentation/TechnicalDesignDocument.tex

```diff
@@ -180,7 +180,7 @@
     The location where the M-block is placed is worked out in another user story: \href{\clickup{2e294}}{\#2e294}.
 
     \myparagraph{\href{\clickup{2e2bq}}{\#2e2bq} As a MGD or player I want to be able to delete a 3D object from the world.}
-    Object Deletion is handled through an InputController, which libgdx uses to catch events from pointer behaviour.
+    Object deletion is handled through an InputController, which libgdx uses to catch events from pointer behaviour.
```

Comment:
It would be nice if class names and other code examples would by in another font or in monospace. You can use \code{mytexthere} if you define it like this: \def\code#1{\texttt{#1}} somewhere above in the document.